plugin.video.composite_for_plex/changelog.txt

```diff
@@ -0,0 +1,216 @@
+0.8.0
```

Comment:
not needed, if possible to avoid in the submission i'd appreciate it

Reply:
removed